src/app/tests/suites/certification/Test_TC_WNCV_1_1.yaml

```diff
@@ -19,60 +19,177 @@ config:
+    - label: "Read the global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Write the default values to mandatory global attribute: EventList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "EventList"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back mandatory global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          value: 0
+
+    - label: "Read the global attribute: ClientGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClientGeneratedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label:
+          "Write the default values to mandatory global attribute:
+          ClientGeneratedCommandList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "ClientGeneratedCommandList"
       arguments:
-          value: 32769
+          value: 0
       response:
           error: 1
 
-    - label: "3b: reads back global attribute: FeatureMap"
+    - label: "Reads back mandatory global attribute: ClientGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ClientGeneratedCommandList"
+      response:
+          value: 0
+
+    - label: "Read the global attribute: ServerGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ServerGeneratedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label:
+          "Write the default values to mandatory global attribute:
+          ServerGeneratedCommandList"
+      disabled: true
+      command: "writeAttribute"
+      attribute: "ServerGeneratedCommandList"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back mandatory global attribute: ServerGeneratedCommandList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "ServerGeneratedCommandList"
+      response:
+          value: 0
+
+    - label: "Read the optional global attribute : FeatureMap"
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
+          value: 0
           constraints:
-              type: uint32
-              notValue: 32769
-    ### OPTIONAL GLOBAL Attributes
-    ### None =======================================
+              type: map32
+
+    - label: "Write the default values to optional global attribute: FeatureMap"
+      command: "writeAttribute"
+      attribute: "FeatureMap"
+      arguments:
+          value: 0
+      response:
+          error: 1
+
+    - label: "Reads back optional global attribute: FeatureMap"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          value: 0
```

Comment:
```suggestion
          value: 0
```

This is wrong since a device might declare some field in the feature map, This gonna be a a failure each time with a device

Reply:
As per the Spec the default value is 0 
![image](https://user-images.githubusercontent.com/84442963/143418266-fc0c65e2-298b-44a8-8a71-2a53842d74d5.png)